rust/module.rs

```diff
@@ -321,81 +321,75 @@ struct ModuleInfo {
 impl ModuleInfo {
     fn parse(it: &mut token_stream::IntoIter) -> Self {
         let mut info = ModuleInfo::default();
-        // The first allowed position is 1, to prevent setting something twice.
-        let mut pos = 0u8;
+
+        const EXPECTED_KEYS: &[&str] = &[
+            "type",
+            "name",
+            "author",
+            "description",
+            "license",
+            "alias",
```

Comment:
I do not yet fully understand if alias can be there multiple times. We should evaluate that somedays.

Reply:
Yeah, there are a few things regarding this -- let me create an issue.